lms/djangoapps/grades/subsection_grade.py

```diff
@@ -192,12 +192,16 @@ def _aggregated_score_from_model(grade_model, is_graded):
         """
         score_type = 'graded' if is_graded else 'all'
         grade_object = grade_model
+        earned_value = getattr(grade_object, 'earned_{}'.format(score_type))
```

Comment:
Nit: If you replace `grade_object` with `grade_model` here, and do the same inside your if-block by replacing `grade_object` with `grade_model.override`, you can get rid of the `grade_object` variable.